src/coreclr/jit/block.h

```diff
@@ -703,9 +697,9 @@ struct BasicBlock : private LIR::Range
     void SetCond(BasicBlock* trueTarget)
     {
         assert(trueTarget != nullptr);
-        bbKind       = BBJ_COND;
-        bbTrueTarget = trueTarget;
-        // TODO-NoFallThrough: also set bbFalseTarget
+        bbKind        = BBJ_COND;
+        bbTrueTarget  = trueTarget;
+        bbFalseTarget = bbNext;
```

Comment:
Should this function take a `falseTarget` parameter that is used to set `bbFalseTarget`? Then, callers would be required to set it. Maybe later on when we remove `bbNext` as fall through?

Reply:
That makes sense to me. I'll open a follow-up PR with that refactor; it might help others avoid hitting asserts around `bbFalseTarget` by forcing them to set it when initializing `bbTrueTarget`.

Thank you for the review!